src/routes/globe/(components)/Globe.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="ts">
+    import { onMount } from 'svelte';
+    import createGlobe from 'cobe';
+
+    let canvas: HTMLCanvasElement;
+    let phi = 0;
+
+    onMount(() => {
+        let width = 0;
+        const onResize = () => canvas && (width = canvas.offsetWidth);
+        window.addEventListener('resize', onResize);
+        onResize();
```

Comment:
this can be moved to a `<svelte:window />`, which will take care of removing it for you